backend/modules/embedder/tfy_embeddings.py

```diff
@@ -124,10 +148,21 @@ def _remote_embed(self, texts, query_mode=False):
             backoff_factor=3,
             status_forcelist=(400, 408, 499, 500, 502, 503, 504),
         )
+        print(
+            f"model: {self.model}, endpoint: {self.endpoint}, api_key: {self.tfy_api_key}"
+        )
         payload = {
-            "inputs": texts,
+            "input": texts,
+            "model": {"name": self.model, "parameters": self.model_parameters},
         }
-        response = session.post(self.endpoint, json=payload, timeout=30)
+        response = session.post(
+            self._endpoint,
+            json=payload,
+            headers={
+                "Authorization": f"Bearer {self.tfy_api_key}",
+            },
+            timeout=30,
+        )
         response.raise_for_status()
         embeddings = response.json()
         return embeddings
```

Comment:
Please check the response structure here, I saw a `response["embeddings"]` key access in the other PR